pydicom/tests/test_rawread.py

```diff
@@ -534,3 +534,25 @@ def testExplVRBigEndian_UndefinedLengthSeq(self):
         # They will be converted when asked for. Check some:
         assert 1 == seq[0].BeamNumber
         assert 'Beam 2' == seq[1].BeamName
+
+    def test_explVR_switch_implVR_in_SQ_item(self):
+        """Raw read: Tolerate missing VR in Sequence item"""
+        # issue 1305
+        hexstr = (
+            "30 0a 00 B0"    # (300a, 00b0) Beam Sequence
```

Comment:
BeamSequence seems to be your favorited sequence :)

Reply:
> BeamSequence seems to be your favorited sequence :)

A lot of years in radiotherapy  :wink: